auth/grpctransport/grpctransport.go

```diff
@@ -244,7 +245,7 @@ func dial(ctx context.Context, secure bool, opts *Options) (*grpc.ClientConn, er
 	}
 	if io := opts.InternalOptions; io != nil {
 		tOpts.DefaultEndpointTemplate = io.DefaultEndpointTemplate
-		tOpts.DefaultMTLSEndpoint = io.DefaultMTLSEndpoint
+		tOpts.DefaultMTLSEndpointTemplate = io.DefaultMTLSEndpoint
```

Comment:
@codyoss Because the new internal options "template" is set here from the old external options "endpoint", this means the `internaloption.WithDefaultMTLSEndpointTemplate` field added in https://github.com/googleapis/google-api-go-client/pull/2880 will not be used. Instead, the workaround will be to set a templatized value (something like `https://test.mtls.UNIVERSE_DOMAIN/`) in `internaloption.WithDefaultMTLSEndpoint`. So should I close that PR?

Reply:
Also, to keep things consistent, I could revert `tOpts.DefaultMTLSEndpointTemplate` to `tOpts.DefaultMTLSEndpoint`. The name of the internal option does not matter right now.

Reply:
SGTM

Reply:
Both done, thanks.